ui/app/src/components/Navigation/PathNavigator/utils.ts

```diff
@@ -14,12 +14,33 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { SandboxItem } from '../../../models/Item';
+import { DetailedItem } from '../../../models/Item';
 
-export function isNavigable(item: SandboxItem): boolean {
-  return item.systemType === 'page'
-    || item.systemType === 'component'
+export function isNavigable(item: DetailedItem): boolean {
+  return item.systemType === 'page';
+}
+
+export function isPreviewable(item: DetailedItem): boolean {
+  return item.systemType === 'component'
     || item.systemType === 'asset'
     || item.systemType === 'template'
-    || item.systemType === 'script';
+    || item.systemType === 'script'
+    || item.systemType === 'taxonomy';
+}
+
+export function isFolder(item: DetailedItem): boolean {
+  return item.systemType === 'folder';
+}
+
+export function getLoaderItems(item: DetailedItem): number {
```

Comment:
- getNumOfMenuOptionsForItem
- Loosen type so it can be used for the different item types